docs/widget/web-component.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Web Component"
+---
+
+# Web Component
+
+If your app is not using React, have no fear, the web component version of the Skip:Go Widget is here!
+
+The web component is created with the `@r2wc/react-to-web-component` library.
```

Comment:
@NotJeremyLiu I was the one who wrote this but I think we can remove this line, it's unnecessary implementation detail for the user

Reply:
Removed in https://github.com/skip-mev/skip-go/pull/80/commits/5176914aa9249d09e1490ae6c863d13f3d7f70f4